testhelpers/clock_test.go

```diff
@@ -0,0 +1,319 @@
+package testhelpers_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	th "github.com/filecoin-project/go-filecoin/testhelpers"
+	tf "github.com/filecoin-project/go-filecoin/testhelpers/testflags"
+)
+
+func TestFakeClockAfter(t *testing.T) {
+	tf.UnitTest(t)
+	fc := th.NewFakeSystemClock(time.Now())
```

Comment:
nit: It's probably worth using a constant time value here (and everywhere else we use `time.Now()` in this file) , e.g. `time.Unix(123456789,0)`